packages/circuit-ui/components/CurrencyInput/__snapshots__/CurrencyInput.spec.tsx.snap

```diff
@@ -56,88 +56,92 @@ exports[`CurrencyInput should adjust input padding and suffix width to match cur
   box-shadow: 0 0 0 1px #999;
 }
 
-.circuit-4::-webkit-input-placeholder {
+.circuit-6::-webkit-input-placeholder {
   color: #999;
   -webkit-transition: color 120ms ease-in-out;
   transition: color 120ms ease-in-out;
 }
 
-.circuit-4::-moz-placeholder {
+.circuit-6::-moz-placeholder {
   color: #999;
   -webkit-transition: color 120ms ease-in-out;
   transition: color 120ms ease-in-out;
 }
 
-.circuit-4:-ms-input-placeholder {
+.circuit-6:-ms-input-placeholder {
   color: #999;
   -webkit-transition: color 120ms ease-in-out;
   transition: color 120ms ease-in-out;
 }
 
-.circuit-4::placeholder {
+.circuit-6::placeholder {
   color: #999;
   -webkit-transition: color 120ms ease-in-out;
   transition: color 120ms ease-in-out;
 }
 
-.circuit-4:hover {
+.circuit-6:hover {
   box-shadow: 0 0 0 1px #666;
 }
 
-.circuit-4:focus {
+.circuit-6:focus {
   box-shadow: 0 0 0 2px #3063E9;
 }
 
-.circuit-4:active {
+.circuit-6:active {
   box-shadow: 0 0 0 1px #3063E9;
 }
 
-<label
+<div
   class="circuit-0"
-  for="input_4"
 >
-  <span
+  <label
     class="circuit-1"
-  >
-    Amount
-  </span>
-  <div
-    class="circuit-2"
+    for="input_4"
   >
     <span
-      class="circuit-3"
+      class="circuit-2 circuit-3"
     >
-      CHF
+      Amount
     </span>
-    <input
+    <div
       class="circuit-4"
-      id="input_4"
-      inputmode="decimal"
-      placeholder="123,45"
-      type="text"
-      value=""
-    />
-  </div>
-</label>
+    >
+      <span
+        class="circuit-5"
+      >
+        CHF
```

Comment:
Hm, wondering is this would pass a WCAG 2.1 AA audit. I guess the label would be announced "Amount CHF" which kind of makes sense? I'll look into this at some point